src/main/java/io/anserini/kg/FreebaseNode.java

```diff
@@ -122,11 +122,15 @@ public static String normalizeObjectValue(String objectValue) {
         return removeEnclosingQuote(objectValue);
       }
     } else if (type.equals(FreebaseNode.RdfObjectType.TEXT)) {
-      Literal parsedLiteral = NTriplesUtil.parseLiteral(objectValue, valueFactory);
-      if (parsedLiteral.getLanguage().toString().equals(LANG_EN)) {
-        return parsedLiteral.stringValue();
-      }
-      return "";
+      // Previously, we were basically throwing away all non-English text...
```

Comment:
Since we're commenting out the logic in this if and just returning object value and the else does the exact same thing, maybe we can comment out this if entirely.

Reply:
Yup, exactly what I had in mind - in my next iteration I'll add the ability to only index English literals.